### PR TITLE
Surah info SEO

### LINKED ISSUE #698: SurahInfo for SEO

### PR DESCRIPTION
**Pull request by mmahalwy**
Fixes #698

### CODE REVIEW

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32784

File: src/server/config/sitemap.js [hunk 1]
```diff
@@ -30,6 +30,12 @@ export default (server) => {
           changefreq: 'weekly',
           priority: 1
         });
+
+        urls.push({
```
Comment (mmahalwy) [inline src/server/config/sitemap.js]: ah cool! Didn't even know!

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32786

File: src/server/config/sitemap.js [hunk 2]
```diff
@@ -30,6 +30,12 @@ export default (server) => {
...
+          url: `/${chapter.chapterNumber}/info`,
```
Comment (naveed-ahmad) [inline src/server/config/sitemap.js]: Keys conversion isn't happening here, not sure why. Switch to `chapter.chapter_number` or `chapter.id`

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32787

**Comment by mmahalwy** [comment]
rebuild

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32788

**Pull request merged**